fiddler/tests/test_fiddler.py

```diff
@@ -1,15 +1,31 @@
 from typing import Any, Callable, Dict
 
+import pytest
+
 from datadog_checks.base import AgentCheck
 from datadog_checks.base.stubs.aggregator import AggregatorStub
 from datadog_checks.dev.utils import get_metadata_metrics
 from datadog_checks.fiddler import FiddlerCheck
 
+from .conftest import FIDDLER_API_KEY
+
 
+# Minimal test
 def test_check(dd_run_check, aggregator, instance):
     # type: (Callable[[AgentCheck, bool], None], AggregatorStub, Dict[str, Any]) -> None
     check = FiddlerCheck('fiddler', {}, [instance])
+    # Prevent the integration from failing
+    with pytest.raises(Exception) as e:
+        dd_run_check(check)
+    assert 'Authorization Required' in str(e.value)
```

Comment:
Without a valid API key, running the check is expected to fail, so we catch the exception and assert the expected error message in the Exception

Reply:
Later on, you can mock fiddler-client responses or add response fixtures to make the tests more robust

Reply:
Sure.